### DISCUSSION TITLE
Konke Motion Sensor 2AJZ4KPBS

### DISCUSSION THREAD

**Discussion by Morphy99** [Koenkk/zigbee2mqtt#6128] [Devices]
![image](https://user-images.githubusercontent.com/5853979/107121775-26c3ac00-688c-11eb-9506-a2f05de699d3.png)
[**Konke 2AJZ4KPBS Motion Sensor**](https://www.zigbee2mqtt.io/devices/2AJZ4KPBS.html#konke-2ajz4kpbs)

I love the form factor of this tiny motion sensor. It sucks that it's limited to channels 15, 20 and 25 but it performs pretty well apart from one annoying issue I'm seeing. False positives. I keep getting notifications when I'm not at home of it triggering. This doesn't happen too often as due to the pandemic, we're always at home but I'm worried when things get back to normal if this thing is going to keep triggering. 

I'd like to find out if anyone else has seen this and what firmware you have on the sensor. Maybe I will try get in touch with Konke to see if there is a fix or a way to reduce sensitivity.

I'm running 190829 on mine.

**Comment by amelchio** [comment]
I am also seeing false positives, enough to keep me from buying more of these. We have the same firmware.

**Reply by Morphy99** [reply]
Thanks for the post! Not good news then. Have you found any that are more reliable and a nice form factor/price?

**Reply by DubhAd** [reply]
Firmware 190829 on mine, I'm not seeing any false positives at all

**Reply by Morphy99** [reply]
Cheers I wonder why we are seeing them and you're not. I've got in touch with Konke and they have asked what hub I'm using as the fw version should be v1.4 but Z2M only reports the build date for some reason. Maybe we need to get this attribute into Z2M, I'm not sure. Could also be something to do with the hardware revision maybe. I've asked what the build date of v1.4 is.

![image](https://user-images.githubusercontent.com/5853979/114436733-9a9c7080-9bbd-11eb-9277-43849165328f.png)

**Reply by etatto** [reply]
I have 4 of them, and I am also not seen any false positive.
Perhaps it may be the environment where the motion sensor is installed, as PIR sensors are sensitive to light and heat. So if the sensor faces a bright window (no curtain) or if it is near a heater, it might affect its detection capabilities.

**Reply by Morphy99** [reply]
Yea I thought that. It's above a window so no direct light or much heat this time of year. But I'll try moving it anyway. Out of curiosity, are your sensors all on the same firmware as mine?

**Reply by etatto** [reply]
Yes, sorry I forget to say it: same firmware 190829 on my four.

**Reply by mbrackjr** [reply]
I have the same FW and sporadic false alarms, especially during very early hours of the morning (say between 05.30 and 06.15).  Any news/updates on this?
